src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,46 @@
+/* eslint-disable max-len */
+
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ movie }) => (
+  <div className="card" data-cy="Movie">
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          data-cy="MovieImage"
+          src={movie.imgUrl}
```

Comment:
it's better to use destructuring assignment to get all properties you need from the `movie` object
by the way, you can even do nested destructuring assignment in arguments